Update for Step3-3

e-yang08 opened the pull request:
## What
<!--- Write the change being made with this pull request --->

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

e-yang08 committed:
Update for Step 3-3

KengoA reviewed (approved):
LGTM! You'll be working on using the database so this won't be necessary but maybe it's a good idea to first check if there's already an existing items.json file, leading to two cases when you set up a new database (json) or append to the existing one.

note: you might want to include the `__pycache__` directory to your .gitignore

e-yang08 committed:
Update for Step 3-4

e-yang08 committed:
Update main.py

e-yang08 committed:
Merge branch 'main' into step3-3

e-yang08 committed:
Check the file path and existence

e-yang08 committed:
Update main.py

e-yang08 committed:
Merge branch 'step3-3' of https://github.com/e-yang08/mercari-build-training-2022 into step3-3

e-yang08 committed:
Update main.py

Create `items.json` if not exists and initialize the file

e-yang08 committed:
Delete python/__pycache__ directory

e-yang08 commented:
> LGTM! You'll be working on using the database so this won't be necessary but maybe it's a good idea to first check if there's already an existing items.json file, leading to two cases when you set up a new database (json) or append to the existing one.
> 
> note: you might want to include the `__pycache__` directory to your .gitignore

Thank you so much for review!

- Updated `main.py` so that it checks if `items.json` file exists and initializes its format to account for the `post` endpoint where it requires dict key `items`.

- Updated `.gitignore` to ignore `__pycache__ directory`

manami-bunbun commented on python/main.py:
```diff
@@ -4,36 +4,85 @@
 def root():
-    return {"message": "Hello, world!"}
+    return {"message": "Welcome to Mercari Items Database"}
+
```
Lovely!

manami-bunbun reviewed (approved):
LGTM!
